promql/promqltest/testdata/histograms.test

```diff
@@ -93,16 +93,16 @@ eval instant at 50m histogram_quantile(0.8, testhistogram_bucket)
 	{start="negative"} 0.3
 
 # More realistic with rates.
-eval instant at 50m histogram_quantile(0.2, rate(testhistogram_bucket[5m]))
-	{start="positive"} 0.048
+eval instant at 50m histogram_quantile(0.2, rate(testhistogram_bucket[10m]))
+    {start="positive"} 0.048
```

Comment:
Your editor has replaced the tab with spaces here. Let's keep things as they are (especially because the next line still has a tab), see also changes below that are only whitespace changes.

Reply:
My bad. Corrected.